squid/src/main.ts

```diff
@@ -70,19 +59,22 @@ const processor = new SubstrateBatchProcessor()
   .setBlockRange({
     from: Number(env.blockstart)
   })
-  .addCall('Proxy.proxy', dataCall)
-  .addCall('Proxy.remove_proxies', dataCall)
-  .addCall('Multisig.as_multi', dataCall)
-  .addCall('Multisig.approve_as_multi', dataCall)
-  .addCall('Multisig.cancel_as_multi', dataCall)
-  .addCall('Multisig.as_multi_threshold_1', dataCall)
-  .addEvent('Proxy.PureCreated', dataEvent)
-  .addEvent('Proxy.AnonymousCreated', dataEvent)
-  .addEvent('Proxy.ProxyAdded', dataEvent)
-  .addEvent('Proxy.ProxyRemoved', dataEvent)
-
-export type Item = BatchProcessorItem<typeof processor>
-export type Ctx = BatchContext<Store, Item>
+  .setFields({ ...fields })
+  .addCall({
+    name: [
```

Comment:
We can move it to the constant

Reply:
totally. Done in [e8a2fde](https://github.com/ChainSafe/Multix/pull/393/commits/e8a2fdee272d99330a1ec517d0c904dca1c2092f)